src/models/contact.js

```diff
@@ -0,0 +1,67 @@
+import { UUIDV4 } from "sequelize";
+
+const contact = (sequelize, DataTypes) => {
+    // Defining our contact table and setting Contact object.
+    const Contact = sequelize.define('contact', {
+        id: {
+            type: DataTypes.UUID,
+            unique: true,
+            primaryKey: true,
+            defaultValue: UUIDV4
+        },
+        user_id: {
+            type: DataTypes.STRING,
+            references: {
+                model: 'users', // 'users' refers to table name
+                key: 'email', // 'id' refers to column email in persons table
+            }
+        },
+        name : {
+            type: DataTypes.STRING,
+            unique: true,
+            required: true
+        },
+        phone: {
+            type: DataTypes.STRING,
+            unique: true,
+            required: true
+        },
+        entity_id: {
+            type: DataTypes.UUID,
+        },
+        email: {
+            type: DataTypes.STRING,
```

Comment:
This, along with the email, should be `DataTypes.JSON` so they can have more than one phone number or email associated with their contact info. The JSON should probably also indicate which is primary contact method. So maybe for email:
```
[
  {
    "email": "example@example.com"
    "isPrimary": true
    }
]
```